jobs/prometheus/spec

```diff
@@ -7,6 +7,7 @@ packages:
 templates:
   bin/prometheus_ctl: bin/prometheus_ctl
   config/prometheus.yml: config/prometheus.yml
+  config/custom.alerts: config/custom.alerts
```

Comment:
Can you please rename `alerts` to `rules` everywhere? We can also configure here [recording rules](https://prometheus.io/docs/querying/rules/).